omod/src/main/java/org/openmrs/module/idgen/web/controller/IdgenEditPatientIdentifiersController.java

```diff
@@ -79,7 +79,7 @@ public void editPatientIdentifiers(ModelMap model, HttpServletRequest request, H
  		// Default Patient Identifiers To Display
     	List<Map<String, Object>> defaultIdentifiers = new ArrayList<Map<String, Object>>();
     	if (patientId != null) {
-    		Patient patient = Context.getPatientService().getPatient(patientId);
```

Comment:
What is the error here that we are trying to correct?

Reply:
Per the other core ticket you just reviewed, there are cases where you get to this page passing in a "patientId" that is actually a personId that is being promoted (but not yet being saved) as a patientId.  In that case, this method call fails, as there are no patients with that ID.  So this fix is acknowledging that the way this page is used is such that patientId could refer to a patientId or a personId and we need to handle that.